server/controllers/aidComments.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable camelcase */
+/* eslint-disable sort-keys */
+import {
+    INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND, getStatusText
+} from 'http-status-codes';
+
+const db = require('./promise').AidDb;
+
+const averageRating = async aid => {
```

Comment:
destructure `aid`

Reply:
I don't understand why i should destructure aid

Reply:
I think **destructuring** `aid` rating into a `const` will throw an error because rating is re-assigned on line 18.
You don't need to `await parseInt(commentTotal / comments.length, 10)` on line 17. These are not async operations.
You could simply do `aid.rating = parseInt(commentTotal / comments.length, 10)`, since the `ratingAverage` isn't used anywhere else.